index.js

```diff
@@ -102,9 +132,37 @@ const closeButton = () => {
 };
 
 let isAscending = true;
+const findRecipe = () => {
+  const searchKey = document.getElementById("searchkey").value.toLowerCase().trim();
+  console.log("Search Key:", searchKey);
+
+  // Filter recipes based on search key
+  const filteredRecipes = searchKey === ''
+    ? recipeObject
+    : recipeObject.filter(recipe => recipe.title.toLowerCase().includes(searchKey));
+  
+  console.log("Filtered Recipes:", filteredRecipes);
+
+  // Display the filtered recipes
+  displayRecipes(filteredRecipes);
 
+  console.log("this");
```

Comment:
Don't forget to clean up your PR from debug code before submitting it.

Reply:
thank you for suggestion